pkg/registry/apis/dashboard/register.go

```diff
@@ -53,6 +65,11 @@ var (
 	_ builder.APIGroupRouteProvider    = (*DashboardsAPIBuilder)(nil)
 )
 
+const (
+	DASHBOARD_SPEC_TITLE            = "title"
+	DASHBOARD_SPEC_REFRESH_INTERVAL = "refresh"
```

Comment:
NIT, Go uses CamelCase for constants:
```suggestion
	DashboardSpecTitle           = "title"
	DashboardSpecRefreshInterval = "refresh"
```